Library/ViewModels/ManagePledgeSummaryViewModelTests.swift

```diff
@@ -33,17 +33,24 @@ final class ManagePledgeSummaryViewModelTests: TestCase {
   }
 
   func testTextOutputsEmitTheCorrectValue() {
-    let backing = .template
-      |> Backing.lens.sequence .~ 999
-      |> Backing.lens.pledgedAt .~ 1_568_666_243.0
-      |> Backing.lens.amount .~ 30.0
-      |> Backing.lens.shippingAmount .~ 7
+    let data = ManagePledgeSummaryViewData(
```

Comment:
I wonder if we should consider creating templates for these view-specific-models 🤔

Reply:
Sure - but not in this PR 😄